model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -179,6 +183,21 @@ message JobState {
 }
 
 
+message GetJobMetricsRequest {
+  string job_id = 1; // (required)
+}
+
+message GetJobMetricsResponse {
+  MetricsResults metrics = 1;
+}
+
+// All metrics for a given job.  Runners may support one or the other or both.
```

Comment:
Today, runners either populate ["attempted" only] or both; dunno if worth clarifying here.

Reply:
It's acceptable to populate committed only as well.